apps/system/js/card.js

```diff
@@ -36,6 +36,9 @@
   Card.prototype = Object.create(BaseUI.prototype);
   Card.prototype.constructor = Card;
 
+  // The point at which we take a swipe to be intentional
+  Card.prototype.SWIPE_WOBBLE_THRESHOLD = 8;
```

Comment:
recently discovered that it's worth doing a `* window.devicePixelRatio` for those constants